src/sentry/utils/flag.py

```diff
@@ -0,0 +1,80 @@
+# Thread safe flag tracking wrapper.
+from contextvars import ContextVar
+
+flag_manager = ContextVar("flag_manager")
+
+
+def initialize_flag_manager(capacity: int = 10) -> None:
```

Comment:
capacity here means we'll keep up to 10 flags in memory at any time?

Reply:
Yes.

Reply:
Starting conservatively? How about 100 😬 ?

Reply:
I know but I'd rather the experiment to be low risk.  Its 10 flags * the number of tasks.  I'm not sure how many that is at the moment.

Reply:
fwiw uwsgi is probably only running 16 threads so I'm being overly safe.

Reply:
Worth documenting the thinking that led to 10. I believe we'll often under-report flags with this value so if the idea is to start at a lower value to reduce risk, then increase eventually that makes sense to me. If that's the case please clarify as a comment on the code or the PR (or even on this thread)